layouts/community/60_ansi/stanrc85-ansi/readme.md

```diff
@@ -0,0 +1,17 @@
+Stanrc85's Standard ANSI 60% Layout
```

Comment:
This is going to bug me. 

This should be:
```suggestion
# Stanrc85's Standard ANSI 60% Layout
```

And the following line could be removed

Reply:
Fixed